docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+db:
+  image: postgres
+web:
+  image: abiosoft/caddy
+  volumes:
+    - ./dist:/srv
+  ports:
+    - "8080:2015"
+  links:
+    - db
+  environment:
+    DATABASE_USER: postgres
+    DATABASE_NAME: postgres
+    DATABASE_HOST: db
+frontend:
+  build: .
+  dockerfile: Dockerfile.yarn
+  env_file:
+    - .env
+  volumes:
+    - .:/usr/src/app
+    - /usr/src/app/node_modules
+  command: yarn watch
+api:
+  image: golang
```

Comment:
Eventually, I think we'll want this pulled out into a separate Dockerfile (which will handle dependency installation).